src/lib/source-handlers/github/get-default-branch.ts

```diff
@@ -1,21 +1,24 @@
 import { Octokit } from '@octokit/rest';
 import * as debugLib from 'debug';
+import type { Target } from '../../types';
 import { getGithubToken } from './get-github-token';
 import { getGithubBaseUrl } from './github-base-url';
 
 const debug = debugLib('snyk:get-github-defaultBranch-script');
 
 export async function getGithubReposDefaultBranch(
-  RepoName: string,
+  target: Target,
   host?: string,
 ): Promise<string> {
   const githubToken = getGithubToken();
   const baseUrl = getGithubBaseUrl(host);
   const octokit: Octokit = new Octokit({ baseUrl, auth: githubToken });
 
-  debug(`fetching the default branch for repo: ${RepoName}`);
-
-  const response = await octokit.request(`GET /repos/${RepoName}`);
+  debug(`fetching the default branch for repo: ${target.owner}/${target.name}`);
 
+  const response = await octokit.repos.get({
```

Comment:
Are we sure that these values will always be there?

Reply:
for github yes